Microsoft.Toolkit.Uwp/Helpers/BackgroundTaskHelper.cs

```diff
@@ -43,6 +43,12 @@ public static bool IsBackgroundTaskRegistered(Type backgroundTaskType)
         /// <returns>Background Task that was registered with the system</returns>
         public static async Task<BackgroundTaskRegistration> RegisterAsync(Type backgroundTaskType, IBackgroundTrigger trigger, bool enforceConditions = true, params IBackgroundCondition[] conditions)
         {
+            // Check if the task is already registered.
```

Comment:
if you return the already registered BackgroundTask like a new BackgroundTask built by BackgroundTaskBuilder, then the code will act like the trigger is added to the BackgroundTask (Actually not).